tdxhydrorapid/inputs.py

```
@@ -115,6 +125,89 @@ def rapid_master_files(streams_gpq: str,
+                dissolve_lake_dict[outlet]['inside'].extend(list(inside_streams))
+            else:
+                dissolve_lake_dict[outlet] = {'inlets': list(inlets), 'inside': list(inside_streams)}
+
+            sgdf.loc[sgdf[id_field].isin(inlets), ds_id_field] = outlet
+
+        # We need to check that no outlets are lakes to delete
+        outlets = set(dissolve_lake_dict.keys())
+        for outlet in outlets:
+            if outlet in streams_to_delete:
+                # Find the real outlet
+                for key, items in dissolve_lake_dict.items():
```

Ruff

Loop control variable `key` not used within loop body

(B007)